packages/docusaurus-plugin-content-blog/src/pluginOptionSchema.ts

```diff
@@ -38,6 +38,7 @@ export const DEFAULT_OPTIONS: PluginOptions = {
   routeBasePath: 'blog',
   path: 'blog',
   editLocalizedFiles: false,
+  authorListPath: 'authors.yml',
```

Comment:
Also not a fan of `authorListPath` naming as the file contains a map not a list. 
The new name should rather be consistent with the other name we have in the code (which should not be `authorList`)